### ISSUE TITLE
Bug encountered when resuming LyCORIS training with dropout settings

### ISSUE THREAD

**Issue by jihnenglin** [KohakuBlueleaf/LyCORIS#129]
I encountered a bug when I attempted to resume training from a previously saved state that utilized dropout when running Kohya's training script. The output images from the new LyCORIS model are very broken. However, the output images appear normal when I utilize `network_module="networks.lora"`, or use `network_weight="path/to/file"` instead of `resume="path/to/state"`.

Here are my settings:

`resume="path/to/state"`
`network_module="lycoris.kohya", network_args=["algo=lora"], network_dropout=0.5, scale_weight_norms=1`
`unet_lr=1e-4, text_encoder_lr=5e-5, optimizer_type="Lion", lr_scheduler="constant"`

**Comment by KohakuBlueleaf** [comment]
I don't know how kohya implement resume.
But I think this is more like a bug in kohya's code (or, something wrong with the way kohya implemented resume)

Since LyCORIS doesn't maintain any trainer. I will close this issue until any explicit problem found in LyCORIS codebase.
